samples/inputs/date-time-input/min-max-value/src/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import { IgrIcon, IgrIconModule, IgrDateTimeInput, IgrDateTimeInputModule, DatePart } from 'igniteui-react';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+
+IgrIconModule.register();
+IgrDateTimeInputModule.register();
+
+export default class DateTimeInputMinMaxValue extends React.Component<any, any> {
+
+    private dateTimeInput: IgrDateTimeInput;
+    constructor(props: any) {
+        super(props);           
+        this.iconRef = this.iconRef.bind(this);     
+        this.dateTimeInputRef = this.dateTimeInputRef.bind(this);
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className="container sample">
+                <IgrDateTimeInput ref={this.dateTimeInputRef} max={new Date(2024, 6, 25)}>
```

Comment:
The sample uses the `min` and `max` properties while the code snippet included in the [docs](https://github.com/IgniteUI/igniteui-xplat-docs/pull/1154/files#diff-3405d20657edffbb2aa13af8509825cd27411739d8d0770f2922a8105468f83cR214-R224) shows the `minValue` and `maxValue`.

```tsx
public dateTimeInputRef(input: IgrDateTimeInput) {
    if (!input) { return; }
    input.minValue = new Date(2021, 0, 1);
}
```

```tsx
 <IgrDateTimeInput ref={this.dateTimeInputRef} maxValue={new Date(2024, 6, 25)}></IgrDateTimeInput>
```

Reply:
The `maxValue` and `minValue` properties are deprecated and `max` and `min` should be used instead. I will update the code snippet to use `max` and `min`.